content/en/logs/log_configuration/sensitive_data_detection.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sensitive Data Detection
+kind: documentation
+beta: true
+further_reading:
+- link: "/security/logs/"
+  tag: "Documentation"
+  text: "Security"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Log Explorer"
+---
+
+<div class="alert alert-info">This page is about Sensitive Data Scanner, which is in currentl in private beta. Contact <a href="https://www.datadoghq.com/support/">Datadog Support</a> if you want to learn more or participate in the beta program. </div>
+
+## Overview
+
+Sensitive data, such as US Social Security Numbers (SSN), credit card numbers, bank routing numbers, API keys, etc., are often exposed unintentionally in application logs and trace events, which can expose your organization to financial and privacy risks.
+
+Often businesses are required to identify, remediate, and prevent the exposure of such sensitive data within their logs due to organizational policies, compliance requirements, industry regulations, and privacy concerns. This is especially true for industries such as banking, financial services, healthcare, and insurance among others.
+
+## Sensitive data scanner
+
+Sensitive data scanner is a stream-based, pattern matching service that you can use to identify, tag, and optionally redact or hash sensitive data. With implementation, your security and compliance teams can introduce a new line of defense in preventing sensitive data from leaking outside your organization.
+
+Datadog's Scanner Library offers a wide variety of rules for commonly detected patterns such as email addresses, SSN, credit card numbers, API keys, authorization tokens, and others.
+
+{{< img src="dashboards/high-density-mode.png" alt="The high-density mode display"  style="width:90%;">}}
```

Comment:
@pgkamat Hi there! Completed an editorial review and the content LGTM. Let me know when the new images are in and I can do a final review!